From: Liang Yan <lyan@suse.com>
Subject: kABI workaround for struct kvm_device
Patch-mainline: Never, kABI workaround
References: git-fixes

Replace a field to  by the patch 
patches.suse/KVM-Allow-kvm_device_ops-to-be-const.patch

Signed-off-by: Liang Yan <lyan@suse.com>
---
 include/linux/kvm_host.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/include/linux/kvm_host.h
+++ b/include/linux/kvm_host.h
@@ -1268,7 +1268,11 @@ extern unsigned int halt_poll_ns_grow_st
 extern unsigned int halt_poll_ns_shrink;
 
 struct kvm_device {
+ #ifndef __GENKSYMS__
 	const struct kvm_device_ops *ops;
+#else
+	struct kvm_device_ops *ops;
+#endif
 	struct kvm *kvm;
 	void *private;
 	struct list_head vm_node;
